Repositories/OrdersRepository.cs

```diff
@@ -12,13 +12,15 @@ public class OrdersRepository : BaseRepository<Order>
         public override Order? Find(params int[] ids)
         {
             return _dbSet.Include(e => e.Products).ThenInclude(e => e.Product).ThenInclude(e => e.Discounts).ThenInclude(e => e.Discount)
-                .Include(e => e.ServiceSlots).ThenInclude(e => e.Service).ThenInclude(e => e.Discounts).ThenInclude(e => e.Discount).FirstOrDefault(e => e.Id == ids[0]);
+                .Include(e => e.ServiceSlots).ThenInclude(e => e.Service).ThenInclude(e => e.Discounts).ThenInclude(e => e.Discount)
+                .Include(e => e.Customer).FirstOrDefault(e => e.Id == ids[0]);
```

Comment:
I probably missed some discussion on why it's done like this - why can't you just use `FirstOrDefault` and query by ID, without joining (I assume thats what `.Include` does) all these tables?

Reply:
Include fills all the relational properties (if Order has Products, then without include there would be no products when getting the entity) this would be fine if we just took the id and got all the entities from their services, but out logic is already built to handle things this way.